RussianFilter/sections/specific.txt

```diff
@@ -22,6 +22,7 @@ Ly9hbi55YW5kZXgucnUv
 doefiratv.net##div[style="display: table;width: 100%;height: 360px;background-color: #f1f1f1;"]
 brestcity.com##.BlockTop1
 brestcity.com###page > div#wrapper~a[rel="nofollow"] > img
+mysitem.ru##.custom
```

Comment:
It seems that it hides content on main page - https://mysitem.ru/
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/61699568-6aeb6c00-ad3b-11e9-8a1e-46805b30d042.png)

</details>

Maybe it would be better to add:
`mysitem.ru##.entry-content > .custom`
or
`mysitem.ru##.entry-content > .custom:has(.adsbygoogle)`